packages/editor/src/state/callbacks.ts

```diff
@@ -363,20 +405,12 @@ export const useDownloadPdf = () =>
     }
   });
 
-export const useDuplicate = () =>
-  useRecoilCallback(({ set }) => () => {
-    set(workspaceMetadataSelector, (state: WorkspaceMetadata) => ({
-      ...state,
-      location: { kind: "local", saved: true } as WorkspaceLocation,
-      id: uuid(),
-    }));
-  });
-
 // returns true if there are no unsaved changes
 export const isCleanWorkspace = (workspace: Workspace): boolean => {
   if (
-    workspace.metadata.location.kind === "local" &&
-    !workspace.metadata.location.saved
+    (workspace.metadata.location.kind === "stored" &&
+      !workspace.metadata.location.saved) ||
+    workspace.metadata.location.kind == "local"
```

Comment:
Just making sure: this means that any unsaved non-example workspace is considered "clean"?

Reply:
Example workspaces (both edited and unedited) are considered clean
Saved workspaces with no unsaved changes are considered clean

Unsaved workspaces (i.e. if you click "new workspace" work in it and don't save it ever) are considered unclean (this is the workspace.meta.location.kind == "local" check)
Unsaved workspaces marked as "stored" that have unsaved changes are considered unclean

Does this clarify?